server/repository/repository.go

```diff
@@ -1,11 +1,14 @@
 package repository
 
 import (
+	"errors"
 	"fmt"
 	"path"
 	"path/filepath"
 	"reflect"
 
+	"github.com/argoproj/argo-cd/util/kustomize"
```

Comment:
nitpick: move import down to all argo-cd related imports.

Reply:
Can we use `goimports` please?